index.js

```diff
@@ -3,7 +3,47 @@
 module.exports = () => {
 
    return {
-      postcssPlugin: 'postcss-var-fallback',
+      postcssPlugin: 'postcss-css-var-fallback',
+
+
+      Root(root) {
+         root.walkDecls((decl) => {
+
```

Comment:
Sorry for the nit, but I'd lose one of these extra lines

Reply:
No ofcourse, fixing now